heltour/tournament/models.py

```diff
@@ -904,7 +904,10 @@ def agreed_to_tos(self):
         )
 
     def get_access_token(self):
-        return self.oauth_token.access_token
+        if self.oauth_token is not None:
+            return self.oauth_token.access_token
+        else:
+            return None
```

Comment:
nit: I slightly prefer this style of guard

```suggestion
        if self.oauth_token is None:
            return None
        return self.oauth_token.access_token
```